File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/model/PerfData.java
package de.dimedis.mobileentry.model;

import de.dimedis.mobileentry.ConfigPref;
import de.dimedis.mobileentry.MobileEntryApplication;
import de.dimedis.mobileentry.backend.response.Border;
import de.dimedis.mobileentry.util.ConfigPrefHelper;

public class PerfData {

    private String deviceSuid;
    private String userSession;
    private String userSuid;
    private String userName;
    private String lang;

    private String fair;
    private String border;
    private String commKey;


    public PerfData() {
    }

    public PerfData initFromPref() {
        Border border = ConfigPrefHelper.getUsersBorder();
        ConfigPref pref = MobileEntryApplication.getConfigPreferences();
        if (border != null) {
            setFair(border.getFair());
            setBorder(border.getBorder());
        }

        if (pref != null) {
            setDeviceSuid(pref.deviceSuid());
            setUserSession(pref.userSession());
            setUserSuid(pref.userSuid());
            setUserName(pref.userName());
            setLang(pref.currentLanguage());
            setCommKey(pref.commKey());
        }
        return this;
    }


    public String getDeviceSuid() {
        return deviceSuid;
    }

    public void setDeviceSuid(String deviceSuid) {
        this.deviceSuid = deviceSuid;
    }

    public String getUserSession() {
        return userSession;
    }

    public void setUserSession(String userSession) {
        this.userSession = userSession;
    }

    public String getUserSuid() {
        return userSuid;
    }

    public void setUserSuid(String userSuid) {
        this.userSuid = userSuid;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getLang() {
        return lang;
    }

    public void setLang(String lang) {
        this.lang = lang;
    }

    public String getFair() {
        return fair;
    }

    public void setFair(String fair) {
        this.fair = fair;
    }

    public String getBorder() {
        return border;
    }

    public void setBorder(String border) {
        this.border = border;
    }

    public String getCommKey() {
        return commKey;
    }

    public void setCommKey(String commKey) {
        this.commKey = commKey;
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/model/PerfData.kt
package de.dimedis.mobileentry.model

import de.dimedis.mobileentry.ConfigPref
import de.dimedis.mobileentry.util.ConfigPrefHelper

class PerfData {
    var deviceSuid: String? = null
    var userSession: String? = null
    var userSuid: String? = null
    var userName: String? = null
    var lang: String? = null
    var fair: String? = null
    var border: String? = null
    var commKey: String? = null
    fun initFromPref(): PerfData {
        val border = ConfigPrefHelper.getUsersBorder()
        val pref: ConfigPref? = ConfigPref
        if (border != null) {
            setFair(border.fair)
            setBorder(border.border)
        }
        if (pref != null) {
            setDeviceSuid(pref.deviceSuid)
            setUserSession(pref.userSession)
            setUserSuid(pref.userSuid)
            setUserName(pref.userName)
            setLang(pref.currentLanguage)
            setCommKey(pref.commKey)
        }
        return this
    }

    @JvmName("getDeviceSuid1")
    fun getDeviceSuid(): String? {
        return deviceSuid
    }

    @JvmName("setDeviceSuid1")
    fun setDeviceSuid(deviceSuid: String?) {
        this.deviceSuid = deviceSuid
    }

    @JvmName("getUserSession1")
    fun getUserSession(): String? {
        return userSession
    }

    @JvmName("setUserSession1")
    fun setUserSession(userSession: String?) {
        this.userSession = userSession
    }

    @JvmName("getUserSuid1")
    fun getUserSuid(): String? {
        return userSuid
    }

    @JvmName("setUserSuid1")
    fun setUserSuid(userSuid: String?) {
        this.userSuid = userSuid
    }

    @JvmName("getUserName1")
    fun getUserName(): String? {
        return userName
    }

    @JvmName("setUserName1")
    fun setUserName(userName: String?) {
        this.userName = userName
    }

    @JvmName("getLang1")
    fun getLang(): String? {
        return lang
    }

    @JvmName("setLang1")
    fun setLang(lang: String?) {
        this.lang = lang
    }

    @JvmName("getFair1")
    fun getFair(): String? {
        return fair
    }

    @JvmName("setFair1")
    fun setFair(fair: String?) {
        this.fair = fair
    }

    @JvmName("getBorder1")
    fun getBorder(): String? {
        return border
    }

    @JvmName("setBorder1")
    fun setBorder(border: String?) {
        this.border = border
    }

    @JvmName("getCommKey1")
    fun getCommKey(): String? {
        return commKey
    }

    @JvmName("setCommKey1")
    fun setCommKey(commKey: String?) {
        this.commKey = commKey
    }
}